src/main/java/com/example/springbootbookshop/dto/UserDto.java

```diff
@@ -0,0 +1,8 @@
+package com.example.springbootbookshop.dto;
+
+public record UserDto(Long id,
```

Comment:
What difference between UserDto and UserRegisterResponseDto? Why do you need both?